lib/parseJson.js

```diff
@@ -82,7 +71,16 @@ function traverseAst(node, callback) {
     }
 }
 
-function fixLocation({node, code}) {
+function fixPosition(loc) {
+    if (loc.line === 1 && !loc.fixed) {
```

Comment:
I am not so happy to monkey-patch an object that the lib does not own.

But the performance fix is more important and for now its okay.

Reply:
One could traverse the tree again after to cleanup, or use a Set to track fixed objects (if that is performant enough)

Reply:
Using the Set strategy will be a minor perf decrease. Test with the offset-position cache (Map) resulted in a ~100ms difference for a 40k loc JSON file compared to native ESLint (~3.5s). 

The current version works and is fast, we have enough unit tests to catch an issue when updating the dependency.